src/main/scala/fs2/kafka/KafkaAdminClient.scala

```diff
@@ -113,6 +128,32 @@ sealed abstract class KafkaAdminClient[F[_]] {
 }
 
 object KafkaAdminClient {
+
+  sealed abstract class CreateTopic[F[_]] {
+
+    /** Name of the topic being created. */
+    def topicName: String
```

Comment:
I added this to make error reporting easier if you've got a collection of `CreateTopic[F]`s